src/Stryker.Core/Stryker.Core/SyntaxHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Stryker.Core
+{
+    // extension methods for SyntaxNode(s)
+    internal static class SyntaxHelper
+    {
+        public static bool IsString(this ExpressionSyntax node)
```

Comment:
I think `IsStringLiteral` is a better name. This does not reassure that the Expression is a string. An variableNameExpression can be of type string too.

Reply:
Fair point, naming is incorrect here. I was not really happy with IsStringLiteral, as the check is broader.
I settled for _IsAStringExpression_ which is very close to the intent: is the expression of a string type, so we do not try arithmetic mutant on it.